frontend/src/metabase-lib/parameters/utils/parameter-values.js

```diff
@@ -9,7 +9,27 @@ import {
 export const PULSE_PARAM_EMPTY = null;
 export const PULSE_PARAM_USE_DEFAULT = undefined;
 
-export function getValuePopulatedParameters(parameters, parameterValues) {
+export function getValuePopulatedParameters(
+  parameters,
+  parameterValues,
+  collectionPreview,
+) {
+  // when it's collection preview (pinned question), we cannot use URL to get
+  // parameter values without value populated in other cases `value: null` will
+  // be handled on BE
+  if (collectionPreview) {
+    return parameterValues
+      ? parameters.map(parameter => {
+          return parameter.id in parameterValues
+            ? {
+                ...parameter,
+                value: parameterValues[parameter.id],
+              }
+            : parameter;
+        })
+      : parameters;
+  }
```

Comment:
What I don't understand, is why if we just sent `undefined` instead of `null` as it happens now in [line 35](https://github.com/metabase/metabase/pull/39072/files#diff-8b5188e613c51e0dd17810bc8859b25544494e8b319ecdb428038048fe4b0542R35) the default value isn't picked up by the BE?

@calherries I thought that sending `undefined` means "use default" but here we send empty parameter list instead.

Reply:
The discussion has moved here: https://github.com/metabase/metabase/pull/39106/files#r1502499549
Not related to or blocking this PR